src/components/Accounts/Actions/FeeFineActions.js

```diff
@@ -644,10 +644,12 @@ class Actions extends React.Component {
     const currentFeeFineType = feefines.find(({ feeFineType }) => feeFineType === account?.feeFineType);
     const currentOwnerId = servicePointOwnerId || currentFeeFineType?.ownerId || account?.ownerId;
     const currentOwner = owners.find(o => o.id === currentOwnerId) || {};
+    const accountTypes = this.props.accounts.map(a => a.feeFineType);
+    const checkSomeNotify = feefines.filter(feeFine => feeFine.actionNoticeId && accountTypes.includes(feeFine.feeFineType));
     const initialValues = {
       ownerId: currentOwnerId,
       amount: calculateSelectedAmount(this.props.accounts),
-      notify: !!(currentFeeFineType?.actionNoticeId || currentOwner?.defaultActionNoticeId),
+      notify: actions.transferMany ? checkSomeNotify.length > 0 : !!(currentFeeFineType?.actionNoticeId || currentOwner?.defaultActionNoticeId),
```

Comment:
The nested conditions within the true/false conditions of the ternary are pretty brutal to wrap your head around. It works, but it might be easier to parse with something like this: 
```
let notify = false;
if (actions.transferMany) {
  if (checkSomeNotify.length > 0) {
    notify = true;
  }
} else {
  if (!!(currentFeeFineType?.actionNoticeId || currentOwner?.defaultActionNoticeId)) {
    notify = true;
  }
}